### ISSUE TITLE
补全失效

### ISSUE THREAD

**Issue by LOST0LOSER** [MicrosoftDocs/intellicode#206]
Issue Type: <b>Bug</b>

After using vscode for a period of time, the code intelligent completion fails somehow. It seems that this situation may be triggered after using the debug function several times

Extension version: 1.2.9
VS Code version: Code 1.46.1 (cd9ea6488829f560dc949a8b2fb789f3cdc05f5d, 2020-06-17T21:13:20.174Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-8100 CPU @ 3.60GHz (4 x 3600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.95GB (15.17GB free)|
|Process Argv||
|Screen Reader|no|
|VM|40%|
</details>

**Comment by markw-t** [comment]
Sorry to hear you're experiencing this problem. We'd like to understand a few more details to help understand what is going on:

1. What language are you using when you see these symptoms? We support Python, Java, TypeScript and JavaScript and TSQL in VS Code, but it will help to understand which one shows the problem.
2. When the problem happens, do you lose all autocompletions (IntelliSense dropdowns) when typing code in the affected language, or just the "starred" recommendation completions at the top of the list, that IntelliCode provides?
3. When the problem occurs, do you see any entries appear in the VS Code IntelliCode output window log? It would be great to see those as they can help us diagnose the issue.  You can get to this in Visual Studio Code by selecting View-Output from the menus, and then selecting VS IntellICode from the dropdown once the Output pane is open. 
Thanks!
Mark
/cc: @osobeso @catycaldwell

**Comment by LOST0LOSER** [comment]
Sorry, I finally found out that the problem may be related to Deno
extension. At that time, I used typescript. Maybe the extensions I
installed had some conflicts. However, it is still worth mentioning that
when I using Python several times, the autocompletion will be invalid after
using Debugger for many times. If this kind of situation occurs again, I
will writeback to you again
Mark Wilson-Thomas <notifications@github.com> 于2020年7月2日周四 上午5:01写道：
